src/tokenizer.ts

```diff
@@ -187,6 +189,7 @@ export namespace Token {
   export function fromKeyword(text: string): Token {
     switch (text) {
       case "abstract": return Token.ABSTRACT;
+      case "and": return Token.AND;
```

Comment:
Is it still TypeScript then?

Reply:
no, and it breaks a test

export function and(loLeft: u32 ...
   and,
   ~~~
 in i64-polyfill.ts(16,3)

Let me know if you ever consider creating a *new* language, I love the clarity of your code.

Reply:
The breakage is probably due to missing entries in `Token.isAlsoIdentifier`. Though, adding non-TS-compatible logical operators doesn't feel right to me.